lib/logger.js

```diff
@@ -103,9 +106,35 @@ class Logger extends EventEmitter {
     let mac = undefined
     let ip = undefined
 
+    if (has(options, 'levels')) {
+      if (!Array.isArray(options.levels)) {
+        const err = new TypeError('levels must be an array')
+        err.meta = {
+          got: typeof options.levels
```

Comment:
this on you might actually want to use `stdlib/type-of` nulls will show up as `object` which is really confusing.

Reply:
fixed